bodhi/server/validators.py

```diff
@@ -14,6 +14,7 @@
 
 from datetime import datetime, timedelta
 
+from cornice.validators import _colander
```

Comment:
Oof. Private APIs :(

Reply:
@jeremycline Perhaps it will assuage your fears to know that I've only used this private API temporarily until there is an upstream release that has the ```_generate_colander_validator()``` that I wrote below.

I've pitched that code upstream and I have the PR approved, but it needs some tests:

https://github.com/Cornices/cornice/pull/465

So it's only a private API because I wanted to use the exact same thing that will eventually be in an upstream Cornice release.

Reply:
Oh and actually, the public API that will ultimately be upstream will actually be the ```colander_querystring_validator()``` that is made by ```_generate_colander_validator()```. So once this is upstream, I will remove the code from this file and just use it directly from cornice.